using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -514,38 +495,38 @@ func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive respo
 {% endtab %}
 {% endtabs %}
 
-### Deleting mobile properties in Experience Platform Launch
+### Deleting mobile properties in the Data Collection UI
 
 {% hint style="danger" %}
-Deleting your property in Experience Platform Launch might cause disruption to your recurring push and in-app messaging activities.
+Deleting your property in the Experience Platform Data Connection UI might cause disruption to your recurring push and in-app messaging activities.
 {% endhint %}
 
-In Experience Platform Launch, if you delete your mobile property, review your mobile property status in the Campaign Standard extension and ensure that the property displays an updated **Deleted in Launch** status. For more information about deleting a property, see [Delete a Property](https://docs.adobe.com/content/help/en/launch/using/reference/admin/companies-and-properties.html#delete-a-property).
+In the Data Collection UI, if you delete your mobile property, review your mobile property status in the Campaign Standard extension and ensure that the property displays an updated **Deleted in ???** status. For more information about deleting a property, please read the [delete a property](https://experienceleague.adobe.com/docs/experience-platform/tags/admin/companies-and-properties.html?lang=en#delete-a-property) section within the Data Collection UI documentation.
```

Comment:
Deleting a property on the Data Collection UI still shows a "Deleted in Launch" in the ACS UI.